server/jetstream_cluster_test.go

```diff
@@ -2995,19 +2995,15 @@ func TestJetStreamClusterPeerOffline(t *testing.T) {
 		var found bool
 		for _, s := range ml.JetStreamClusterPeers() {
 			if s == rs.Name() {
-				if shouldBeOffline {
```

Comment:
I think we need to keep this, however, maybe move this for-loop, etc.. inside the checkFor() (and therefore replace t.Fatalf() with return fmt.Errorf()).

Reply:
That just checks that if we expect it to be there its there immediately regardless of offline status.

Reply:
I added this test yesterday. The idea of `shouldBeOffline` means that I want to fail if I find it in the list, because it should not be. The issue maybe is that it should be retried (hence the move to checkFor).
Removing this code means that you are going to fail only if it should be present and is not, but will not capture the fact that it is present when it should not.